cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+
+**The key thing to be aware of** is that not all AIOs are created equally
+in terms of absorption. It is _very important_ to understand what the material
+composition of these diapers is so that you know what you're dealing with.
+
+For example:
+[Modern Cloth Nappies AIOs](https://www.modernclothnappies.co.uk/collections/pearl-pocket-all-in-one-nappies/products/pearl-pocket-all-in-one-reusable-one-size-nappy)
+(my absolute favourite AIOs from our stash), have a super absorbent
+hemp cotton mix insert.
+<img width="533" alt="image" src="https://user-images.githubusercontent.com/4185328/170531761-98c62f2c-a3ac-4eab-b941-04940c289e2f.png">
+
+The [Bambino Miosolo AIOs](https://www.bambinomio.com/collections/miosolo/products/miosolo-diaper)
+sit at the complete opposite end of that scale, with a microfibre core which,
+despite the claims on the website of 'super absorbency', has _very little
+absorbency_ and _very good marketing_.
+
+I wouldn't risk using one of these without reinforcement at any time of day.
+Why do we still have them you ask? Because they have a _great_ fit and are
+very practical in that sense!
+
+<img width="492" alt="image" src="https://user-images.githubusercontent.com/4185328/170533205-f0cfd3c2-b9f6-49a2-8f9c-702ea9f271cf.png">
+
+
+### 2. Pocket Diapers
+We started using One Size diapers in the Spring and AIOs were a firm favourite
+because as I mentioned, the gave me very little work to put together.
+
+But then Winter came [insert your favourite Game of Thrones meme here].
+
+I struggled to get anything to dry, even with a dehumidifier on constantly.
+So we moved to pocket diapers for Winter.
+
+They take a LOT more time to put together but the shells air dry very quickly
+and if you're desperate you can throw the inserts in the tumble dryer too.
+
+We use _high quality_ natural material inserts.
+
+Our ***hemp inserts*** are triple layer hemp/cotton inserts with a _high_ percentage
+of hemp (55% hemp, 45% cotton): https://www.eco-tiny.com/product/absorvente-de-canhamo-e-algodao-organico-ecotiny
+
+Our ***bamboo inserts*** are 100% bamboo
+[Little Lamb bamboo doublers](https://littlelamb.com/products/bamboo-booster-doubler).
+
+Our infallible inserts combinations here are:
++ Morning diapers: 2 hemp inserts + 1 bamboo doubler folded in half (giving
+us 4 layers of bamboo booster) in the front of the diaper
++ Afternoon diapers: 1 hemp insert + 1 bamboo doubler folded in half (giving
+us 4 layers of bamboo booster) in the front of the diaper
+
+> We use our pockets throughout the Winter (with AIOs filling in the gaps)
+and whenever we are travelling for ease of washing and drying.
+
+
+### 3. Fitted diapers with covers (night time)
+For night time we use fitted diapers which are made entirely of absorbent material
+(in our case usually 100% bamboo or a bamboo/hemp mix), reinforced
+with hemp or bamboo inserts and covered with a PUL cover or wrap to make them
```

Comment:
"**PUL**" is only given as an abbreviation. what does it _mean_? 🤷‍♂️

Reply:
#36